### ISSUE TITLE
Make error - docker

### ISSUE THREAD

**Issue by CarlosLannister** [BBVA/deeptracy#98]
After make start, following error appear

Pulling deeptracy-buildbot (deeptracy-buildbot:)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

**Comment by nilp0inter** [comment]
Try executing `make down` before `make`.

**Comment by CarlosLannister** [comment]
New error Makefile:7: recipe for target 'image' failed

Installing dependencies from Pipfile.lock (bb0834)…
An error occurred while installing zope.interface==4.5.0 ; python_version != '3.2.*' and python_version != '3.1.*' and python_version >= '2.7' and python_version != '3.0.*' --hash=sha256:21506674d30c009271fe68a242d330c83b1b9d76d62d03d87e1e9528c61beea6 --hash=sha256:3d184aff0756c44fff7de69eb4cd5b5311b6f452d4de28cb08343b3f21993763 --hash=sha256:467d364b24cb398f76ad5e90398d71b9325eb4232be9e8a50d6a3b3c7a1c8789 --hash=sha256:57c38470d9f57e37afb460c399eb254e7193ac7fb8042bd09bdc001981a9c74c --hash=sha256:9ada83f4384bbb12dedc152bcdd46a3ac9f5f7720d43ac3ce3e8e8b91d733c10 --hash=sha256:a1daf9c5120f3cc6f2b5fef8e1d2a3fb7bbbb20ed4bfdc25bc8364bc62dcf54b --hash=sha256:e6b77ae84f2b8502d99a7855fa33334a1eb6159de45626905cb3e454c023f339 --hash=sha256:e881ef610ff48aece2f4ee2af03d2db1a146dc7c705561bd6089b2356f61641f --hash=sha256:f41037260deaacb875db250021fe883bf536bf6414a4fd25b25059b02e31b120! Will try again.
Installing initially failed dependencies…
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/site-packages/pipenv/core.py", line 1874, in do_install
[pipenv.exceptions.InstallError]:       keep_outdated=keep_outdated
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/site-packages/pipenv/core.py", line 1253, in do_init
[pipenv.exceptions.InstallError]:       pypi_mirror=pypi_mirror,
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/site-packages/pipenv/core.py", line 859, in do_install_dependencies
[pipenv.exceptions.InstallError]:       retry_list, procs, failed_deps_queue, requirements_dir, **install_kwargs
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/site-packages/pipenv/core.py", line 763, in batch_install
[pipenv.exceptions.InstallError]:       _cleanup_procs(procs, not blocking, failed_deps_queue, retry=retry)
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/site-packages/pipenv/core.py", line 681, in _cleanup_procs
[pipenv.exceptions.InstallError]:       raise exceptions.InstallError(c.dep.name, extra=err_lines)
[pipenv.exceptions.InstallError]: ['Collecting zope.interface==4.5.0 (from -r /tmp/pipenv-tn05fgi7-requirements/pipenv-n0q_oe1p-requirement.txt (line 1))', '  Using cached https://files.pythonhosted.org/packages/a8/1c/ebc5de89de058c7aaf8f5d1013020273449d7069bb4036287c52cea9a25f/zope.interface-4.5.0-cp36-cp36m-manylinux1_x86_64.whl']
[pipenv.exceptions.InstallError]: ['THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.', '    zope.interface==4.5.0 from https://files.pythonhosted.org/packages/a8/1c/ebc5de89de058c7aaf8f5d1013020273449d7069bb4036287c52cea9a25f/zope.interface-4.5.0-cp36-cp36m-manylinux1_x86_64.whl#sha256=727aa737a48dd33d6859296f15edb54e85ccdfa5667513f7a50daf362b3df75b (from -r /tmp/pipenv-tn05fgi7-requirements/pipenv-n0q_oe1p-requirement.txt (line 1)):', '        Expected sha256 21506674d30c009271fe68a242d330c83b1b9d76d62d03d87e1e9528c61beea6', '        Expected     or 3d184aff0756c44fff7de69eb4cd5b5311b6f452d4de28cb08343b3f21993763', '        Expected     or 467d364b24cb398f76ad5e90398d71b9325eb4232be9e8a50d6a3b3c7a1c8789', '        Expected     or 57c38470d9f57e37afb460c399eb254e7193ac7fb8042bd09bdc001981a9c74c', '        Expected     or 9ada83f4384bbb12dedc152bcdd46a3ac9f5f7720d43ac3ce3e8e8b91d733c10', '        Expected     or a1daf9c5120f3cc6f2b5fef8e1d2a3fb7bbbb20ed4bfdc25bc8364bc62dcf54b', '        Expected     or e6b77ae84f2b8502d99a7855fa33334a1eb6159de45626905cb3e454c023f339', '        Expected     or e881ef610ff48aece2f4ee2af03d2db1a146dc7c705561bd6089b2356f61641f', '        Expected     or f41037260deaacb875db250021fe883bf536bf6414a4fd25b25059b02e31b120', '             Got        727aa737a48dd33d6859296f15edb54e85ccdfa5667513f7a50daf362b3df75b']
ERROR: ERROR: Package installation failed...
The command '/bin/sh -c pipenv install --system --deploy' returned a non-zero code: 1
Makefile:7: recipe for target 'image' failed
make: *** [image] Error 1

**Comment by nilp0inter** [comment]
Are you using the latest version of the repository? Try a fresh:

```bash
git clone https://github.com/BBVA/deeptracy.git
cd deeptracy
make down
make
```

**Comment by nilp0inter** [comment]
Also, are you using any kind of Pypi proxy?

**Comment by CarlosLannister** [comment]
Just make a new test in a different machine and same error

Installing dependencies from Pipfile.lock (bb0834)…
An error occurred while installing zope.interface==4.5.0 ; python_version != '3.2.*' and python_version != '3.1.*' and python_version >= '2.7' and python_version != '3.0.*' --hash=sha256:21506674d30c009271fe68a242d330c83b1b9d76d62d03d87e1e9528c61beea6 --hash=sha256:3d184aff0756c44fff7de69eb4cd5b5311b6f452d4de28cb08343b3f21993763 --hash=sha256:467d364b24cb398f76ad5e90398d71b9325eb4232be9e8a50d6a3b3c7a1c8789 --hash=sha256:57c38470d9f57e37afb460c399eb254e7193ac7fb8042bd09bdc001981a9c74c --hash=sha256:9ada83f4384bbb12dedc152bcdd46a3ac9f5f7720d43ac3ce3e8e8b91d733c10 --hash=sha256:a1daf9c5120f3cc6f2b5fef8e1d2a3fb7bbbb20ed4bfdc25bc8364bc62dcf54b --hash=sha256:e6b77ae84f2b8502d99a7855fa33334a1eb6159de45626905cb3e454c023f339 --hash=sha256:e881ef610ff48aece2f4ee2af03d2db1a146dc7c705561bd6089b2356f61641f --hash=sha256:f41037260deaacb875db250021fe883bf536bf6414a4fd25b25059b02e31b120! Will try again.
Installing initially failed dependencies…
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/site-packages/pipenv/core.py", line 1874, in do_install
[pipenv.exceptions.InstallError]:       keep_outdated=keep_outdated
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/site-packages/pipenv/core.py", line 1253, in do_init
[pipenv.exceptions.InstallError]:       pypi_mirror=pypi_mirror,
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/site-packages/pipenv/core.py", line 859, in do_install_dependencies
[pipenv.exceptions.InstallError]:       retry_list, procs, failed_deps_queue, requirements_dir, **install_kwargs
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/site-packages/pipenv/core.py", line 763, in batch_install
[pipenv.exceptions.InstallError]:       _cleanup_procs(procs, not blocking, failed_deps_queue, retry=retry)
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/site-packages/pipenv/core.py", line 681, in _cleanup_procs
[pipenv.exceptions.InstallError]:       raise exceptions.InstallError(c.dep.name, extra=err_lines)
[pipenv.exceptions.InstallError]: ['Collecting zope.interface==4.5.0 (from -r /tmp/pipenv-wmo0bsv6-requirements/pipenv-_5wql087-requirement.txt (line 1))', '  Using cached https://files.pythonhosted.org/packages/a8/1c/ebc5de89de058c7aaf8f5d1013020273449d7069bb4036287c52cea9a25f/zope.interface-4.5.0-cp36-cp36m-manylinux1_x86_64.whl']
[pipenv.exceptions.InstallError]: ['THESE PACKAGES DO NOT MATCH THE HASHES FROM THE REQUIREMENTS FILE. If you have updated the package versions, please update the hashes. Otherwise, examine the package contents carefully; someone may have tampered with them.', '    zope.interface==4.5.0 from https://files.pythonhosted.org/packages/a8/1c/ebc5de89de058c7aaf8f5d1013020273449d7069bb4036287c52cea9a25f/zope.interface-4.5.0-cp36-cp36m-manylinux1_x86_64.whl#sha256=727aa737a48dd33d6859296f15edb54e85ccdfa5667513f7a50daf362b3df75b (from -r /tmp/pipenv-wmo0bsv6-requirements/pipenv-_5wql087-requirement.txt (line 1)):', '        Expected sha256 21506674d30c009271fe68a242d330c83b1b9d76d62d03d87e1e9528c61beea6', '        Expected     or 3d184aff0756c44fff7de69eb4cd5b5311b6f452d4de28cb08343b3f21993763', '        Expected     or 467d364b24cb398f76ad5e90398d71b9325eb4232be9e8a50d6a3b3c7a1c8789', '        Expected     or 57c38470d9f57e37afb460c399eb254e7193ac7fb8042bd09bdc001981a9c74c', '        Expected     or 9ada83f4384bbb12dedc152bcdd46a3ac9f5f7720d43ac3ce3e8e8b91d733c10', '        Expected     or a1daf9c5120f3cc6f2b5fef8e1d2a3fb7bbbb20ed4bfdc25bc8364bc62dcf54b', '        Expected     or e6b77ae84f2b8502d99a7855fa33334a1eb6159de45626905cb3e454c023f339', '        Expected     or e881ef610ff48aece2f4ee2af03d2db1a146dc7c705561bd6089b2356f61641f', '        Expected     or f41037260deaacb875db250021fe883bf536bf6414a4fd25b25059b02e31b120', '             Got        727aa737a48dd33d6859296f15edb54e85ccdfa5667513f7a50daf362b3df75b']
ERROR: ERROR: Package installation failed...
The command '/bin/sh -c pipenv install --system --deploy' returned a non-zero code: 1
Makefile:7: recipe for target 'image' failed
make: *** [image] Error 1

**Comment by nilp0inter** [comment]
You are absolutely right. I wasn't hitting this error because I had cached docker layers preventing `make` of rerunning the `pipenv` command. I removed all my docker cache and now I can replicate it.

Apparently the error is caused by a new distributions of a dependency package for the same version that already exist (zopefoundation/zope.interface#124). That was confusing pipenv locking mechanism.

I will update the Pipenv.lock file soon.

Thank you!
